### ISSUE TITLE
Add support for `prefixItems`

### ISSUE THREAD

**Issue by edobrb** [kogosoftwarellc/open-api#864]
From what I understand, OAS 3.1 should support the same validations offered by JSON Schema. It would therefore be possible to define a type `{ type: 'array', minItems: 2, maxItems: 2, prefixItems: [...] }` that would be used to define tuples.

[https://json-schema.org/draft/2020-12/json-schema-validation.html](https://json-schema.org/draft/2020-12/json-schema-validation.html)

I hope I'm not mistaken, but currently it is not supported in this repository:
https://github.com/kogosoftwarellc/open-api/blob/c805d4333c79891c29f8567dfc7f08da6334a64b/packages/openapi-types/index.ts#L148-L151

**Comment by saschahofmann** [comment]
Although I cant find it in this codebase when using the package I can find this now 
``` ts 
 interface ArraySchemaObject extends BaseSchemaObject {
        type: ArraySchemaObjectType;
        items: ReferenceObject | SchemaObject;
        prefixItems: ReferenceObject | SchemaObject;
    }
```
which seems wrong to me? Shouldn't prefixItems be an array? Also not sure how it ends up in my nodeModules like this. I cant find `prefixItems` mentioned anywhere.

**Comment by edobrb** [comment]
Yes `prefixItems` should be an array. See `TupleSchemaObject` in #865
